GameData/RealismOverhaul/RO_RecommendedMods/Procedurals/RO_ProceduralParts.cfg

```diff
@@ -379,7 +379,7 @@
 	%RSSROConfig = True
 
 	@title = Procedural Tank (Service Module)
```

Comment:
Maybe change the title to:
@title = Procedural Service Module
Then lower  down in the description it could work better with:
@description = <b><color=red>Not intended to hold fuel</color></b>, these specialized modules hold batteries, payloads and life support tanks in a controlled environment. Insulation and high pressure make service modules more expensive, heavier and with a much lower useful volume than conventional tanks.